zed_wrapper/launch/include/zed_camera.launch.py

```diff
@@ -202,6 +205,10 @@ def generate_launch_description():
                 'xacro_path',
                 default_value=TextSubstitution(text=default_xacro_path),
                 description='Path to the camera URDF file as a xacro file.'),
+            DeclareLaunchArgument(
+                'ros_parameters_path',
+                default_value='',
+                description='The path to an additional parameters file to load into the core node'),
```

Comment:
What do you think about
`description='The path to an additional parameters file to override the parameters with custom settings'),`